### PR TITLE
Adding docs for the bindingKey option in associations

### PR DESCRIPTION
**Pull request by lorenzo**

### CODE REVIEW

File: en/orm/associations.rst [hunk 1]
```diff
@@ -181,10 +181,13 @@ Possible keys for hasOne association arrays include:
   the current model. If you're defining a 'User hasOne Address'
   relationship, the className key should equal 'Addresses'.
 - **foreignKey**: the name of the foreign key found in the other
-  model. This is especially handy if you need to define multiple
+  table. This is especially handy if you need to define multiple
   hasOne relationships. The default value for this key is the
   underscored, singular name of the current model, suffixed with
   '\_id'. In the example above it would default to 'user\_id'.
+- **bindingKey**: The name of the column in the currebt table, that will be used
```
Comment (markstory) [inline en/orm/associations.rst]: Typo in current.

**Comment by lorenzo** [comment]
@markstory fixed

File: fr/orm/associations.rst [hunk 2]
```diff
@@ -188,11 +186,14 @@ Les clés possibles pour une association hasOne sont:
 - **className**: le nom de la classe de la table que l'on souhaite associer au
   model actuel. Si l'on souhaite définir la relation 'User a une Address', la
   valeur associée à la clé 'className' devra être 'Addresses'.
-- **foreignKey**: le nom de la clé étrangère que l'on trouve dans l'autre model.
+- **foreignKey**: le nom de la clé étrangère que l'on trouve dans l'autre table.
   Ceci sera particulièrement pratique si vous avez besoin de définir des
   relations hasOne multiples. La valeur par défaut de cette clé est le nom du
   model actuel (avec des underscores) suffixé avec '\_id'. Dans l'exemple
   ci-dessus la valeur par défaut aurait été 'user\_id'.
+- **bindingKey**: le nom de la colone dans la table courante, qui sera utilisée
```
Comment (HavokInspiration) [inline fr/orm/associations.rst]: colonne

File: fr/orm/associations.rst [hunk 3]
```diff
@@ -188,11 +186,14 @@ Les clés possibles pour une association hasOne sont:
...
+  pour correspondre à la ``foreignKey``. S'il n'est pas spécifié, la clé
+  primaire (par exemple la colone id de la table ``Users``) sera utilisée.
```
Comment (HavokInspiration) [inline fr/orm/associations.rst]: colonne*

File: fr/orm/associations.rst (hunk 2 above)
Comment (HavokInspiration) [inline fr/orm/associations.rst]: colonne (The same typo happens a few time below as well)

File: fr/orm/associations.rst [hunk 4]
```diff
@@ -372,9 +376,12 @@ Les clés possibles pour les tableaux d'association hasMany sont:
   (l'User a plusieurs Commentaires), la valeur associée à la clef 'className'
   devra être 'Comment'.
 - **foreignKey**: le nom de la clé étrangère que l'on trouve dans l'autre
-  model. Ceci sera particulièrement pratique si vous avez besoin de définir
+  table. Ceci sera particulièrement pratique si vous avez besoin de définir
   plusieurs relations hasMany. La valeur par défaut de cette clé est le nom
-  du model actuel (avec des underscores) suffixé avec '\_id'
+  du model actuel (avec des underscores) suffixé avec '\_id'.
+- **bindingKey**: le nom de la colone dans lae table courante, qui sera utilisée
```
Comment (HavokInspiration) [inline fr/orm/associations.rst]: la* table courante

**Comment by antograssiot** [comment]
@HavokInspiration merci

**Pull request merged**